auditlog_security/views/auditlog_view.xml

```diff
@@ -0,0 +1,149 @@
+                        <field name="user_id" readonly="1"/>
+                        <field name="old_value" readonly="1"/>
+                        <field name="new_value" readonly="1"/>
+                        <field name="old_value_text" readonly="1"/>
+                        <field name="new_value_text" readonly="1"/>
+                    </group>
+                </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_auditlog_log_line_tree">
+        <field name="name">auditlog.log.line.tree</field>
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+                <tree delete="0">
+                    <field name="create_date"/>
+                    <field name="user_id"/>
+                    <field name="method"/>
+                    <field name="field_name"/>
+                    <!--<field name="old_value"/>-->
+                    <field name="old_value_text"/>
+                    <!--<field name="new_value"/>-->
+                    <field name="new_value_text"/>
+                </tree>
+        </field>
+    </record>
+
+    <record id="audit_log_line_action" model="ir.actions.act_window">
+        <field name="name">View Log Lines</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">auditlog.log.line</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <!-- show logs for partner out side of technical too, now that we have rules
+    for all types of users, will place in "contact" -->
+
+
+    <record id="auditlog_form_view_ext" model="ir.ui.view">
+        <field name="name">auditlog rule form extension</field>
+        <field name="model">auditlog.rule</field>
+        <field name="inherit_id" ref="auditlog.view_auditlog_rule_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[last()]" position="after">
+                <group>
+                    <div>
+                        <p>
+                            Add fields here to make any changes to them (audit log lines)
+                            visible to members of the selected groups.
+                        </p>
+                    </div>
+                </group>
+                <group>
+                    <!-- in the domain are the blacklisted fields in auditlog they
+                        would be skipped anyway, so we repeat here to avoid incongruencies
+                    -->
+                    <field name="auditlog_line_access_rule_ids">
+                        <tree>
+                            <field name="model_id" readonly="1" invisible="1"/>
+                            <field name="group_ids" widget="many2many_tags"
+                                   options="{'no_create': True}"
+                             />
+                            <field name="field_ids"  widget="many2many_tags"
+                                    options="{'no_create': True}"
```

Comment:
I still think there should be a domain here, restricting to model_id

Reply:
the tree in in the formview is not editable, so there is no problem, below you can see the extra domain is there and working on the popup-form.  I will add it for extra robustness.